lib/relation-definition.js

```diff
@@ -2920,9 +2922,8 @@ EmbedsMany.prototype.create = function(targetModelData, options, cb) {
   targetModelData = targetModelData || {};
   cb = cb || utils.createPromiseCallback();
 
-  var embeddedList = this.embeddedList();
-
   var inst = this.callScopeMethod('build', targetModelData);
+  var embeddedList = this.embeddedList();
```

Comment:
Reverse the sequence of defining `inst` and `embeddedList`, so `embeddedList` won't refer to a dangle `[]`.